### ISSUE TITLE
Reduce the size of README

### ISSUE THREAD

**Issue by shajoezhu** [ga4gh/server#315]
The README file is currently too long, propose to 
 - [ ] reduce the size of README 
 - [x] split the installation section to three separate (Deployment, Client tools and Development) documentation pages, and create links to them.

**Comment by shajoezhu** [comment]
i m working on this

**Comment by jeromekelleher** [comment]
This sounds good @shajoezhu. One possible approach is to use Sphinx to generate documentation for us and use http://pythonhosted.org or https://read-the-docs.readthedocs.org to host our HTML documentation. I would probably lean towards read-the-docs as it's a bit more fully featured.

**Comment by jeromekelleher** [comment]
I've created a project for the ga4gh-reference-implementation on read-the-docs.org. If you want to be added to the list of maintainers, sign up for an account and I'll add you.

The documentation will be available at http://ga4gh-reference-implementation.readthedocs.org/

**Comment by lh3** [comment]
A bit off-topic (I can create a new issue if you prefer). Installation seem complicated especially for someone without the root permission. Is it possible to optionally use flask's builtin server just for demonstration? I am imagining an installation procedure like:

```
pip install --user --pre ga4gh
wget http://www.well.ox.ac.uk/~jk/ga4gh-example-data.tar.gz
tar -zxf ga4gh-example-data.tar.gz
ga4gh-server serve ga4gh-example-data/
```
Cassie used to show a BAM-based read server implantation (I can't find the link now). It has an installation procedure something like the above, though I haven't tried it myself.

**Comment by jeromekelleher** [comment]
I completely agree @lh3, thanks for the input. We need to make the distinction between a quick demo (which should be as easy as your example here), and a full production deployment (which does need all the complicated Apache configuration).

We'll try and get this into the docs over the next couple of days.

**Comment by jeromekelleher** [comment]
I'm working on part (1) of this - reduce the size of the README down to something sensible.

**Comment by dcolligan** [comment]
Partially fixed in #318

**Comment by jeromekelleher** [comment]
Fixed in #328
